pkg/infra/usagestats/usage_stats_service_test.go

```diff
@@ -0,0 +1,146 @@
+package usagestats
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/grafana/grafana/pkg/bus"
+	"github.com/grafana/grafana/pkg/services/licensing"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUsageStatsService_GetConcurrentUsersStats(t *testing.T) {
+	sqlStore := sqlstore.InitTestDB(t)
+	uss := &UsageStatsService{
+		Bus:      bus.New(),
+		SQLStore: sqlStore,
+		License:  &licensing.OSSLicensingService{},
+	}
+	t.Run("Get concurrent users stats should return a histogram", func(t *testing.T) {
+		createConcurrentTokens(t, sqlStore)
+
+		ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+		t.Cleanup(func() {
+			cancel()
+		})
+
+		t.Run("Should refresh stats when forced", func(t *testing.T) {
+			actualResult, err := uss.GetConcurrentUsersStats(ctx, true)
+			require.NoError(t, err)
+
+			expectedResult := &ConcurrentUsersStats{
+				BucketLe3:   1,
+				BucketLe6:   2,
+				BucketLe9:   3,
+				BucketLe12:  4,
+				BucketLe15:  5,
+				BucketLeInf: 6,
+			}
+
+			assert.Equal(t, expectedResult, actualResult)
+
+			err = createToken(7, sqlStore)
+			assert.NoError(t, err)
```

Comment:
`assert` would allow the test to continue after an error.
```suggestion
			require.NoError(t, err)
```